src/views/ManageList.jsx

```diff
@@ -1,7 +1,118 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+import toast, { Toaster } from 'react-hot-toast';
+
+export function ManageList({ listPath }) {
+	console.log('listPath:', listPath);
+	const [itemName, setItemName] = useState('');
+	const [itemNextPurchaseTimeline, setItemNextPurchaseTimeline] = useState('');
+
+	const handleItemNameTextChange = (e) => {
+		setItemName(e.target.value);
+	};
+
+	const handleNextPurchaseChange = (e) => {
+		setItemNextPurchaseTimeline(e.target.value);
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		console.log('Submitting item:', {
+			itemName,
+			itemNextPurchaseTimeline,
+			listPath,
+		});
+
+		try {
+			await toast.promise(
+				addItem(listPath, {
+					itemName,
+					daysUntilNextPurchase: itemNextPurchaseTimeline,
+				}),
+				{
+					pending: 'Adding item to list.',
```

Comment:
Something that is nearly universally enforced by a linter, but doesn't appear to be here, is consistency between single and double quotes. I notice some inconsistencies here, which isn't a big deal, but it's usually nice to see picking between one or the other (I'm team double quotes personally) and sticking to it

Reply:
Yeah I could of sworn it was getting linted but here they are inconsistent 😂